src/components/common/CaseDetails/CaseDetails.js

```diff
@@ -49,12 +66,40 @@ function CaseDetails() {
   };
 
   const saveNotesHandler = event => {
-    placeholderArray.push(newNote);
+    //Create a time stamp in MM/DD/YYYY fomate
+    let timeStamp = Date.now();
+    new Intl.DateTimeFormat('en-US', {
+      year: 'numeric',
+      month: '2-digit',
+      day: '2-digit',
+      hour: '2-digit',
+      minute: '2-digit',
+      second: '2-digit',
+    }).format(timeStamp);
```

Comment:
Do you use this timestamp anywhere?  I can't find it.

Reply:
Yeah I meant to delete that but guess I messed it